KsApi/models/graphql/GraphUserCreditCard.swift

```diff
@@ -2,7 +2,7 @@
 
 // TODO: Remove public access
 public struct GraphUserCreditCard: Decodable {
-  public var storedCards: CreditCardConnection
+  public var nodes: [CreditCard]
```

Comment:
We don't need the `CreditCardConnection` class anymore based on the testing. By removing it we no longer need to access the cards from the envelope like this:

`$0.fragments.userFragment.storedCards.storedCards.nodes` and we can eliminate one of those `storedCards` properties.

I also did this because a handful of tests were failing because the decoding was expecting two `storedCards` keys and our response from the server is not structured that way. This is a more accurate representation of what graph is returning.